### PR TITLE
Guess Who- game

### PR DESCRIPTION
**Pull request by malinvan**
guesswhogame-mvan.netlify.app

### CODE REVIEW

**Review by matildasvensson85 — CHANGES_REQUESTED** [review]
Great work! You met all requirements for blue level and most of the requirements for the red level. You added lot of new options to choose from in the dropdown-menu that made the game easier and more fun to play. 

In general I think the code looks great, its indentation is good, its tidy and easy to understand,  the structure is logical and you use variable names in a way that makes the code easy to read. In most cases everything is coherent (some exceptions in the comment I added). You also just used let (vs const) when needed to. Even though you wrote you didn't have time for styling, I see you did some minor changes in the css well. Great work!

File: code/script.js [hunk 1]
```diff
@@ -244,100 +243,132 @@ const generateBoard = () => {
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log(secret);
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/script.js:248]: Yes, remember to always clean your code from all console.log()s and commented out code blocks and tips left for you by teachers 🗡️ ✂️

File: code/script.js [hunk 2]
```diff
@@ -244,100 +243,132 @@ const generateBoard = () => {
...
 }
 
-// This function to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  setSecret();
```
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/script.js:253]: Remember that you can set your VScode to insert all needed semicolons for you. Go to settings, search for a semicolon and choose it 🤖 🦾

File: code/script.js [hunk 3]
```diff
@@ -244,100 +243,132 @@ const generateBoard = () => {
...
+  const selectedElementValue = questions.options[questions.selectedIndex].value;
+  const selectedElementInnerText = questions.options[questions.selectedIndex].innerText;
 
   if (category === 'hair color') {
     currentQuestion = {
       attribute: 'hairColor',
-      // value: ,
-      // 👆 add the value from the input here
+      value: selectedElementValue,
+      innerText: selectedElementInnerText,
```
Comment (KarolCislowski) [inline code/script.js:267]: It is not a good practice to name fields the same way that DOM functions or fields.

**Review by KarolCislowski — APPROVED** [review]
This is a very nice project 🌈 

I know it was an extra point of this assignment to style it your way but remember that there is no another way to master CSS then practice it over and over again 🦾 

Remember that you can always come back to it when you have some spare time and add more functionalities to make it an even greater portfolio asset! 🦄

**Pull request closed without merging**